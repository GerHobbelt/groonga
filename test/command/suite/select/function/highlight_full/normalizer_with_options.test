table_create Entries TABLE_NO_KEY
column_create Entries body COLUMN_SCALAR ShortText

load --table Entries
[
{"body": "Mroonga is a ＭｙＳＱＬ storage engine based on Groonga (ぐるんが). <b>Rroonga</b> is a Ruby binding of Groonga (グルンガ)."}
]

select Entries --output_columns \
  'highlight_full(body, \
                  "NormalizerNFKC121(\\"unify_kana\\", true)", \
                  true, \
                  "ぐるんが", \
                  "<span class=\\"keyword1\\">", \
                  "</span>")'
