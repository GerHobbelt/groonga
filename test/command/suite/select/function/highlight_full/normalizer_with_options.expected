table_create Entries TABLE_NO_KEY
[[0,0.0,0.0],true]
column_create Entries body COLUMN_SCALAR ShortText
[[0,0.0,0.0],true]
load --table Entries
[
{"body": "Mroonga is a ＭｙＳＱＬ storage engine based on Groonga (ぐるんが). <b>Rroonga</b> is a Ruby binding of Groonga (グルンガ)."}
]
[[0,0.0,0.0],1]
select Entries --output_columns   'highlight_full(body,                   "NormalizerNFKC121(\\"unify_kana\\", true)",                   true,                   "ぐるんが",                   "<span class=\\"keyword1\\">",                   "</span>")'
[
  [
    0,
    0.0,
    0.0
  ],
  [
    [
      [
        1
      ],
      [
        [
          "highlight_full",
          null
        ]
      ],
      [
        "Mroonga is a ＭｙＳＱＬ storage engine based on Groonga (<span class=\"keyword1\">ぐるんが</span>). &lt;b&gt;Rroonga&lt;/b&gt; is a Ruby binding of Groonga (<span class=\"keyword1\">グルンガ</span>)."
      ]
    ]
  ]
]
